src/org/labkey/test/components/QueryMetadataEditorPage.java

```diff
@@ -0,0 +1,91 @@
+package org.labkey.test.components;
+
+import org.labkey.test.Locator;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+public class QueryMetadataEditorPage extends DomainDesignerPage
+{
+    public QueryMetadataEditorPage(WebDriver driver)
+    {
+        super(driver);
+    }
+
+    @Override
+    public void clickFinish()
+    {
+        scrollIntoView(elementCache().finishButton());
+        shortWait().until(ExpectedConditions.elementToBeClickable(elementCache().finishButton()));
+        click(Locator.button(elementCache().finishButton().getText()));
+    }
+
+    public void reset()
+    {
+        scrollIntoView(newElementCache().resetButton());
+        shortWait().until(ExpectedConditions.elementToBeClickable(newElementCache().resetButton()));
+        click(Locator.button(newElementCache().resetButton().getText()));
+        click(Locator.button("Reset")); // Reset confirmation on the confirm modal
```

Comment:
Is there any navigation or other indication that the reset is complete?

Reply:
there is no indication saying Reset successful.